src/inmanta/server/services/compilerservice.py

```diff
@@ -648,6 +649,8 @@ async def request_recompile(
             they contain resources that are being exported.
         :param mergeable_env_vars: a set of env vars that can be compacted over multiple compiles.
             If multiple values are compacted, they will be joined using spaces.
+        :param removed_resource_sets: [DEPRECATED] Resource sets marked for removal should be passed by setting the
+            INMANTA_REMOVED_RESOURCE_SET_ID environment variable in the env_vars or the mergeable_env_vars parameter.
```

Comment:
I think we can afford to simply drop this. Afaik the only interface here is between core and inmanta-lsm, which we will upgrade together. @wouterdb do you agree?

Reply:
yes